src/sql/workbench/contrib/notebook/browser/notebookStyles.ts

```diff
@@ -137,7 +148,9 @@ export function registerNotebookThemes(overrideEditorThemeSetting: boolean, conf
 			}
 
 			// Override code editor background if color is defined
-			let codeBackground = inactiveBorder; // theme.getColor(EDITOR_GROUP_HEADER_TABS_BACKGROUND);
+			//let codeBackground = inactiveBorder; // theme.getColor(EDITOR_GROUP_HEADER_TABS_BACKGROUND);
```

Comment:
What happens when themes are changed after these changes?

Reply:
The registerThemingParticipant means that this callback is called every time the theme is change and will cause the new colors for the theme to be applied

Reply:
I added border colors to the sql path colorRegistry and removed all related code from this file.